Frontend/ui-driver/src/Services/APITypes.purs

```diff
@@ -1202,3 +1203,218 @@ instance showGetGetPerformanceRes :: Show GetPerformanceRes where show = generic
 instance standardEncodeGetPerformanceRes :: StandardEncode GetPerformanceRes where standardEncode (GetPerformanceRes req) = standardEncode req
 instance decodeGetPerformanceRes :: Decode GetPerformanceRes where decode = defaultDecode
 instance encodeGetPerformanceRes :: Encode GetPerformanceRes where encode = defaultEncode
+
+--------------------------------------------------- getCategories ----------------------------------------------------
+data GetCategoriesReq = GetCategoriesReq String
+
+newtype GetCategoriesRes = GetCategoriesRes
+  { categories :: Array
+    { label    :: String
+    , logoUrl  :: String
```

Comment:
@ThisIsMani  Do we have logoUrl in resp?

Reply:
Not able to find in swagger

Reply:
Yes, we do. I checked in Postman